### ISSUE TITLE
Install as Nuxt module

### ISSUE THREAD

**Issue by Atinux** [janniks/vue-notion#3]
Hi @janniks, congrats on this great project, I have been looking forward a Notion integration with Vue for so long 😊 

I would be really nice to be able to install it as a nuxt module:

```js
// nuxt.config.js
export default {
  buildModules: ['vue-notion/nuxt']
}
```

The idea would be to have a `nuxt.js` file directory (or `nuxt/index.js`) that:
- Add `vue-notion` to `build.transpile` (https://github.com/janniks/vue-notion/blob/main/example/nuxt.config.js#L35)
- Inject `$notion` helper directly so it can be used

This may help you if you need any help setting a module: https://nuxtjs.org/blog/creating-a-nuxt-module

**Comment by janniks** [comment]
That sounds great! Wasn't sure what the official best way of solving this was. I'd be happy to accept a PR.

Otherwise, I can look into this further in January 🎄

**Comment by janniks** [comment]
Closed in #13 and released in [v0.3.0](https://github.com/janniks/vue-notion/releases/tag/0.3.0)

**Comment by janniks** [comment]
Thanks @Atinux for the help 🙏
